helpers/tercerosMidHelper/propiedad.go

```diff
@@ -32,3 +32,27 @@ func GetCargoFuncionario(id int) (cargo []*models.Parametro, outputError map[str
 
 	return
 }
+
+func GetDocumentoTercero(id int) (documento []*models.DatosIdentificacion, outputError map[string]interface{}) {
```

Comment:
Falta documentar, cambiar `...` en el _suggested change_, digamos "GetDocumentoTercero hace tal cosa"
```suggestion
// GetDocumentoTercero ...
func GetDocumentoTercero(id int) (documento []*models.DatosIdentificacion, outputError map[string]interface{}) {
```